src/scripts/main.js

```diff
@@ -17,20 +17,19 @@ const promise2 = new Promise((resolve, reject) => {
 
 promise1
   .then(() => {
-    const message = document.createElement('div');
-
-    message.classList.add('message');
-    message.textContent = 'Promise was resolved!';
-
-    document.body.append(message);
+    createMessage('message', 'Promise was resolved!');
   });
 
 promise2
   .catch(() => {
-    const message = document.createElement('div');
+    createMessage('error-message', 'Promise was rejected!');
+  });
 
-    message.classList.add('error-message');
-    message.textContent = 'Promise was rejected!';
+function createMessage(clas, mes) {
```

Comment:
```suggestion
function createMessage(class, message) {
```
Не сокращай названия, оставляй их понятными